integration/experiment/gpu_frequency_sweep/gpu_frequency_sweep.py

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env python3
+#
+#  Copyright (c) 2015 - 2022, Intel Corporation
+#  SPDX-License-Identifier: BSD-3-Clause
+#
+
+'''
+Helper functions for running power sweep experiments.
+'''
+
+import sys
+import argparse
+
+import geopmpy.agent
+
+from experiment import launch_util
+from experiment import common_args
+from experiment import machine
+from experiment.frequency_sweep import frequency_sweep
+
+
+def setup_run_args(parser):
+    common_args.setup_run_args(parser)
+    common_args.add_run_max_turbo(parser)
+    common_args.add_min_frequency(parser)
+    common_args.add_max_frequency(parser)
+    common_args.add_step_frequency(parser)
+    parser.add_argument('--max-gpu-frequency', dest='max_gpu_frequency',
+                        action='store', type=float, default=None,
+                        help='top gpu frequency setting for the sweep')
+    parser.add_argument('--min-gpu-frequency', dest='min_gpu_frequency',
+                        action='store', type=float, default=None,
+                        help='bottom gpu frequency setting for the sweep')
+    parser.add_argument('--step-gpu-frequency', dest='step_gpu_frequency',
+                        action='store', type=float, default=None,
+                        help='increment in hertz between gpu frequency settings for the sweep')
+
+
+def setup_gpu_frequency_bounds(mach, min_gpu_freq, max_gpu_freq,
+                                  step_gpu_freq):
+
+    #TODO: Don't hardcode this. What.
```

Comment:
Need to mimic add  `gpu_frequency_*()` functions to machine.py like the `frequency_*()` in there. `GPU_CORE_FREQENCY_MAX_AVAIL` and `GPU_CORE_FREQENCY_MIN_AVAIL` then call those functions here.